// http://github.com/vitax/suckless
--- dwm/dwm.c
+++ dwm/dwm.c
@@ -65,1 +65,1 @@
-enum { SchemeNorm, SchemeSel, SchemeUrg }; /* color schemes */
+enum { SchemeNorm, SchemeSel, SchemeTagSel, SchemeUrg }; /* color schemes */

--- dwm/dwm.c
+++ dwm/dwm.c
@@ -777,3 +777,3 @@
 	for (i = 0; i < LENGTH(tags); i++) {
 		w = TEXTW(tags[i]);
-		drw_setscheme(drw, scheme[m->tagset[m->seltags] & 1 << i ? SchemeSel : SchemeNorm]);
+		drw_setscheme(drw, scheme[m->tagset[m->seltags] & 1 << i ? SchemeTagSel : SchemeNorm]);
